AuroraControlsMaui/ChipGroup.cs

```diff
@@ -117,6 +117,22 @@ public double VerticalSpacing
         set => SetValue(VerticalSpacingProperty, value);
     }
 
+    /// <summary>
+    /// The start offset to insert empty space before the first chip.
+    /// </summary>
+    public static readonly BindableProperty StartOffsetProperty =
```

Comment:
I don't like that this is just for the start. It should be more generic for the start and end. Conceptually, it is fine, but it should apply to both the start and end equally.